src/main/java/core/basesyntax/dao/ConverterDataToObjImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.FruitTransaction;
+import core.basesyntax.db.DataBase;
+import java.util.List;
+
+public class ConverterDataToObjImpl implements ConverterDataToObj {
+    private static final String SPLITERATOR = ",";
+    private static final int OPERATION_SEQUENCE = 0;
+    private static final int FRUIT_NAME_SEQUENCE = 1;
+    private static final int FRUIT_QUANTITY_SEQUENCE = 2;
+
+    @Override
+    public void convertData(String filePath) {
+        ReaderDataFromFileImpl readerDataFromFile = new ReaderDataFromFileImpl();
```

Comment:
SRP violation, don't tightly couple reading and parsing, call services consecutively in Main

Reply:
not fixed